ruleset.xml

```diff
@@ -5,5 +5,7 @@
 	<file>example.php</file>
 	<arg name="report" value="full"/>
 	<arg value="sp"/>
-	<rule ref="WordPress"/>
+	<rule ref="WordPress-Extra">
+		<exclude name="Generic.Files.LineEndings.InvalidEOLChar" />
```

Comment:
Actually true, I'll revert that. It will break on my local machine, but I can use an adjusted ruleset locally.